src/Run/CommandLine.cs

```diff
@@ -1813,7 +1825,7 @@ private static string GetEntryAssemblyName()
         private bool _noDashOnParameterSets = true;
         private bool _noSpaceOnQualifierValues;
         private string[] _parameterSetsWhereQualifiersMustBeFirst;
-        private bool _qualifiersUseOnlyDash = true;
+        private bool _qualifiersUseOnlyDash = false;
```

Comment:
We can chat later.  Basically setting the public property at the earliest chance I can with the way it's currently written will throw.  Most people complaining were more interested in -- versus / anyways so it's fine.